tests/acceptance/search-test.js

```diff
@@ -19,26 +20,13 @@ module('Acceptance | search', (hooks) => {
 		mockAdsService(this.owner);
 	});
 
-	skip('visiting /search', async (assert) => {
-		const searchInput = '.side-search__input';
-		const testQuery = 'test query';
-
-		await visit('/search');
-
-		assert.equal(currentURL(), '/search');
-
-		await fillIn(searchInput, testQuery);
-		await triggerKeyEvent(searchInput, 'keyup', 'Enter');
-
-		assert.equal(currentURL(), '/search?query=test%20query');
+	test('visiting /search', async (assert) => {
```

Comment:
I would actually take this chance to change the test case name. Maybe "Visiting /search results in displaying 4 result cards and a correct title"?